apps/web/src/app/(default)/dashbord/[slug]/_components/remove-all-registrations-button.tsx

```diff
@@ -0,0 +1,67 @@
+"use client";
+
+import { useState } from "react";
+import { Dialog, DialogContent } from "@radix-ui/react-dialog";
+
+import { removeAllRegistrations } from "@/actions/remove-all-registrations";
+import { Button } from "@/components/ui/button";
+
+type RemoveAllRegistrationsButtonProps = {
+  slug: string;
```

Comment:
Ref tidligere kommentar.

```suggestion
  id: string;
```